pandas/tests/test_generic.py

```diff
@@ -996,6 +996,59 @@ def test_describe_percentiles_insert_median(self):
         self.assertTrue('0%' in d1.index)
         self.assertTrue('100%' in d2.index)
 
+    def test_describe_percentiles_unique(self):
+        # GH13104
+        df = tm.makeDataFrame()
+        with self.assertRaises(ValueError):
+            df.describe(percentiles=[0.1, 0.2, 0.4, 0.5, 0.2, 0.6])
+        with self.assertRaises(ValueError):
+            df.describe(percentiles=[0.1, 0.2, 0.4, 0.2, 0.6])
+
+    def test_describe_percentiles_formatting(self):
+        # GH13104
```

Comment:
```python
pd.DataFrame().describe()
Traceback (most recent call last):
  File "/usr/share/python3.5/site-packages/IPython/core/interactiveshell.py", line 3066, in run_code
    exec(code_obj, self.user_global_ns, self.user_ns)
  File "<ipython-input-8-e81e6dc1b4a2>", line 1, in <module>
    pd.DataFrame().describe()
  File "/home/users/piotr/workspace/pandas-pijucha/pandas/core/generic.py", line 4975, in describe
    d = pd.concat(ldesc, join_axes=pd.Index([names]), axis=1)
  File "/home/users/piotr/workspace/pandas-pijucha/pandas/tools/merge.py", line 845, in concat
    copy=copy)
  File "/home/users/piotr/workspace/pandas-pijucha/pandas/tools/merge.py", line 878, in __init__
    raise ValueError('No objects to concatenate')
ValueError: No objects to concatenate

pd.Series().describe()
Out[9]: 
count    0.0
mean     NaN
std      NaN
min      NaN
25%      NaN
50%      NaN
75%      NaN
max      NaN
dtype: float64
```
This one looks nicer:
```python
pd.Series(dtype=object).describe()
Out[10]: 
count     0
unique    0
dtype: int64
```

An empty series has a float dtype by default  - as it''s always considered to be 1d, I think. An empty data frame has no default number of columns and no dtypes at all, so I'm not sure what kind of output `describe()` should show.